lightning/src/util/events.rs

```diff
@@ -147,6 +148,14 @@ pub enum Event {
 #[cfg(test)]
 		error_data: Option<Vec<u8>>,
 	},
+	/// Indicates an outbound MPP payment partially failed. Probably some intermediary node dropped
+	/// something. You may wish to retry this portion of the payment with a different path.
+	MPPFragmentFailed {
```

Comment:
I've seen "part" used frequently. Wondering if we should use similar terminology.